packages/studio-ui/src/web/translations/fr.json

```diff
@@ -683,6 +683,20 @@
     }
   },
   "topNav": {
+    "deploy": {
+      "tooltip": {
+        "deployBot": "Déployer le bot",
+        "lastDeploy": "Dernier déploiement",
+        "needsTraining": "Le bot a besoin d'une formation",
```

Comment:
We more often use the word `entraînement` than `formation` when referring to the word `training`